docs/UserGuide.md

```diff
@@ -160,18 +160,20 @@ Format: `list`
 --------------------------------------------------------------------------------------------------------------------
 ### [**Finding Projects :** `findproj`](#table-of-contents)
 
-Finds all projects whose names contain any of the specified keywords (case-insensitive) and displays them as a list with index numbers.
+Finds all projects whose names that matches any of the specified keywords (case-insensitive) and displays them as a list with index numbers.
```

Comment:
Should be Finds all projects whose names match but I'll change this in next PR since I'll be working on UG too